llvm/test/Transforms/LoopVectorize/conditional-scalar-assignment-vplan.ll

```diff
@@ -0,0 +1,68 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
```

Comment:
@fhahn, you said:

> I think I am missing something, but the non-EVL codegen doens't seem to use and predicated memory accesses, so it should be able to do it with a generic target?

Yes, the non-evl codegen can do it with generic target, but the EVL related RUN line requires us to be target specific. But then we'd have two test files with the same test cases, only differing by RUN lines. I can make that change if you'd prefer. It was my opinion that it is better to not duplicate the test cases and compare differences in vectorization in one file, but if you disagree, I am content to concede.